src/org/sosy_lab/java_smt/test/FloatingPointFormulaManagerTest.java

```diff
@@ -773,24 +774,44 @@ public void ieeeFpConversion() throws SolverException, InterruptedException {
         .isTautological();
   }
 
+  /**
+   * Map the function over the input list and prove the returned assertions.
+   *
+   * @param f A function that takes values from the list and returns assertions
+   * @param args A list of arguments to the function
+   */
+  private <T> void proveForAll(Function<T, BooleanFormula> f, List<T> args)
```

Comment:
just styling and personal preference: switch parameters. Using the list of args upfront makes it sound like `prove for all <numbers> that <function> is satisfiable`.